scripts/emit-rule-sweep.py

```
@@ -0,0 +1,160 @@
+    successIndicators: [{{ responseContains: "egress_attempted" }}]
+    expectedDetections:
+      - {{ ruleID: R0011, ruleName: Unexpected Egress Network Traffic, containerName: {c} }}
+'''
+    return b
+
+
+def main():
+    ap = argparse.ArgumentParser(description="Emit the standard rule-family attack block")
+    ap.add_argument("--container", required=True, help="k8s container name the execs land in")
+    ap.add_argument("--stage", default="/dev/shm", help="writable dir for staged binaries/links")
+    ap.add_argument("--mount", default="/dev/shm", help="a REAL volume mount, for R1004")
```

Ruff (0.16.0)

[error] Probable insecure usage of temporary file or directory: "/dev/shm"

(S108)